presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/ColumnReference.java

```diff
@@ -44,30 +45,42 @@ public ColumnReference(String tableName, String columnName)
     @Override
     public Optional<Symbol> getAssignedSymbol(PlanNode node, Session session, Metadata metadata, SymbolAliases symbolAliases)
     {
-        if (!(node instanceof TableScanNode)) {
+        TableHandle tableHandle;
+        Map<Symbol, ColumnHandle> assignments;
+
+        if (node instanceof TableScanNode) {
+            TableScanNode tableScanNode = (TableScanNode) node;
+            tableHandle = tableScanNode.getTable();
+            assignments = tableScanNode.getAssignments();
+        }
+        else if (node instanceof IndexSourceNode) {
+            IndexSourceNode indexSourceNode = (IndexSourceNode) node;
+            tableHandle = indexSourceNode.getTableHandle();
+            assignments = indexSourceNode.getAssignments();
+        }
+        else {
             return Optional.empty();
         }
 
-        TableScanNode tableScanNode = (TableScanNode) node;
-        TableMetadata tableMetadata = metadata.getTableMetadata(session, tableScanNode.getTable());
+        TableMetadata tableMetadata = metadata.getTableMetadata(session, tableHandle);
         String actualTableName = tableMetadata.getTable().getTableName();
 
         // Wrong table -> doesn't match.
         if (!tableName.equalsIgnoreCase(actualTableName)) {
             return Optional.empty();
         }
 
-        Optional<ColumnHandle> columnHandle = getColumnHandle(tableScanNode.getTable(), session, metadata);
+        Optional<ColumnHandle> columnHandle = getColumnHandle(tableHandle, session, metadata);
 
         checkState(columnHandle.isPresent(), format("Table %s doesn't have column %s. Typo in test?", tableName, columnName));
 
-        return getAssignedSymbol(tableScanNode, columnHandle.get());
+        return getAssignedSymbol(assignments, columnHandle.get());
     }
 
-    private Optional<Symbol> getAssignedSymbol(TableScanNode tableScanNode, ColumnHandle columnHandle)
+    private Optional<Symbol> getAssignedSymbol(Map<Symbol, ColumnHandle> assignments, ColumnHandle columnHandle)
```

Comment:
I think the problem was that I was doing git fetch and not git fetch --all.  Resolved now.